src/main/java/com/yanna/stepanova/model/Book.java

```diff
@@ -9,8 +9,8 @@
 import java.math.BigDecimal;
 import lombok.Data;
 
-@Entity
 @Data
```

Comment:
dont use data with entity as it cause bugs, replace with getter/setter annotations. fix in all models